content/docs/guides/health_checks/control_plane_health_probes.md

```diff
@@ -41,8 +41,8 @@ Events:
   Type     Reason     Age               From               Message
   ----     ------     ----              ----               -------
   Normal   Scheduled  24s               default-scheduler  Successfully assigned osm-system/osm-controller-85fcb445b-fpv8l to osm-control-plane
-  Normal   Pulling    23s               kubelet            Pulling image "openservicemesh/osm-controller:v0.9.2"
-  Normal   Pulled     23s               kubelet            Successfully pulled image "openservicemesh/osm-controller:v0.9.2" in 562.2444ms
+  Normal   Pulling    23s               kubelet            Pulling image "openservicemesh/osm-controller:{{< param osm_full_version >}}"
```

Comment:
Since the versions here are all just examples for sample output and not particularly relevant, we don't need to parameterize these. My only fear is someone will read too far into these and worry if the output isn't exactly the same with the latest version. And it's not worth updating the entire output here manually for every release. When I first added these the version was v0.8.0.